scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -112,15 +112,15 @@ workflow GvsExtractCallset {
 
   call Utils.GetBQTableLastModifiedDatetime as SamplesTableDatetimeCheck {
     input:
-      project_id = project_id,
+      project_id = query_project,
```

Comment:
Line 18, 
`String query_project = project_id`
I think we need to update the upstream method and passing the correct query_project